### PR TITLE
Tried to get as many TBDs gone - and finalize this document for a 1.0 version

### PR DESCRIPTION
**Pull request by dirkx**
Our german peers are really in need of a 1.0 spec - attempt to finalize all loose ends I know of.

### CODE REVIEW

**Review by jschlyter — APPROVED** [review]

File: hcert_spec.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Note that the data structure is of importance here. The actual wire format is la
 
 ### Structure of the Electronic Health Certificate
 
-The HCERT is structured and encoded as a CBOR payload with a COSE digital signature. This is commonly known as a "CBOR Web Token" (CWT), and is defined in [RFC 8392](https://tools.ietf.org/html/rfc8392). The HCERT payloads, as defined below, is transported in a hcert claim (claim key TBD).
+The HCERT is structured and encoded as a CBOR payload with a COSE digital signature. This is commonly known as a "CBOR Web Token" (CWT), and is defined in [RFC 8392](https://tools.ietf.org/html/rfc8392). The HCERT payloads, as defined below, is transported in a hcert claim.
```
Comment (gabywh) [inline hcert_spec.md:38]: payloads -> payload

File: hcert_spec.md [hunk 2]
```diff
@@ -161,14 +161,17 @@ In addition - each DSC certificate:
...
 - MUST contain a 256bit Subject key identifier
 
-TBD.
+### Simplified CSCA/DSC
 
 It is expressly allowed to have the CSCA be identical to the DSC. Or in other words - if a country uses a set of self-signed certificates; it would submit these both as its CSCA’s and as its DSC list.
 
 As of this version of the specifications - countries should NOT assume that any CRL information is used; or that the Private Key Usage Period is verified by implementors.
 
 Instead - the primary validity mechanism is appearance on the most recent version of the list.
```
Comment (gabywh) [inline hcert_spec.md:170]: not clear what this sentence means

File: hcert_spec.md [hunk 3]
```diff
@@ -161,14 +161,17 @@ In addition - each DSC certificate:
...
 
+### ICAO-ML and Trust Centers
+
+Member states can use a sepearate CSCA (as per the WHO advice) - but may also use submit thier existing eMRT CSCA and/or DSC certificates; and may even chose to procure these from (commercial) trustcenters - and submit these. However - any DSC certificate must always be signed by the CSCA submitted by that country.
```
Comment (gabywh) [inline hcert_spec.md:174]: ref to WHO advice?
thier -> their

**Review by gabywh — CHANGES_REQUESTED** [review]
Instead of trying to fix the grammar etc here in review, I'll submit a PR as mod to this doc on a separate branch.

**Review by martin-lindstrom — APPROVED** [review]

**Review by fredriklj — APPROVED** [review]

File: hcert_spec.md (hunk 1 above)
Comment (dirkx) [reply hcert_spec.md:38]: fixed

File: hcert_spec.md (hunk 3 above)
Comment (dirkx) [reply hcert_spec.md:174]: fixed

**Pull request merged**